websrc/gadget_menu/src/gadget_menu_main.tsx

```diff
@@ -914,6 +1009,57 @@ class ControlPanel extends React.Component< {}, ControlPanelState >
 		}
 	}
 
+	@bind
+	private setAutoLaunch( gadgetUrl: string, manifest?: AardvarkManifest )
```

Comment:
enableAutoLaunch/disableAutoLaunch would be more consistent names for these.

Reply:
switched them out for the suggestions